src/assets/Generator.Shared/RequestContentHelper.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Azure.Core
+{
+    internal static class RequestContentHelper
+    {
+        public static RequestContent FromDictionary<T>(IDictionary<string, T> dictionary) where T : notnull
```

Comment:
Could we create an issue to track the format story we discussed offline?

Reply:
I don't think we will take any actions on that format issue until we run into it. So we could open up then.